api/controllers/calendarEntry.controller.ts

```diff
@@ -13,22 +24,49 @@ interface CalendarEntry {
   updatedAt: Date
 };
 
-export const getCalendarEntries = (request: Request, response: Response, next: NextFunction) => {
-  const date = new Date()
-  const entries: CalendarEntry[] = [
+const dayAfter = (date) => (new Date((date).valueOf() + 1000*3600*24))
```

Comment:
I saw in VScode, it suggests that the date param here can be typed too when I hover over it